discord/application_command_create.go

```diff
@@ -107,3 +107,36 @@ func (c MessageCommandCreate) CommandName() string {
 }
 
 func (MessageCommandCreate) applicationCommandCreate() {}
+
+type EntryPointCommandCreate struct {
+	Name                     string                      `json:"name"`
+	NameLocalizations        map[Locale]string           `json:"name_localizations,omitempty"`
+	DefaultMemberPermissions *json.Nullable[Permissions] `json:"default_member_permissions,omitempty"`
+	// Deprecated: Use Contexts instead
+	DMPermission     *bool                         `json:"dm_permission,omitempty"`
+	IntegrationTypes []ApplicationIntegrationType  `json:"integration_types,omitempty"`
+	Contexts         []InteractionContextType      `json:"contexts,omitempty"`
+	NSFW             *bool                         `json:"nsfw,omitempty"`
+	Handler          *EntryPointCommandHandlerType `json:"handler,omitempty"`
```

Comment:
same here, this does not need to be a pointer I think, `omitempty` should omit the 0